kratos/CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ target_compile_definitions(KratosVersion PRIVATE
 
 ## Define library KratosCore to be included in all of the others
 add_library(KratosCore SHARED ${KRATOS_CORE_SOURCES} ${KRATOS_CORE_TESTING_ENGINE_SOURCES} ${KRATOS_CORE_INPUT_OUTPUT_SOURCES} ${KRATOS_TEST_SOURCES} $<TARGET_OBJECTS:KratosVersion>)
-target_link_libraries(KratosCore PUBLIC gidpost triangle)
+target_link_libraries(KratosCore PUBLIC gidpost triangle clipper)
```

Comment:
Is there really no other alternative than compiling this and add it as a dependency in the linking stage?

Reply:
I'd be open for any suggestion?